spec/controllers/builds_controller_spec.rb

```diff
@@ -0,0 +1,65 @@
+require 'rails_helper'
+
+RSpec.describe BuildsController, type: :controller do
+  let(:payload) { File.read('spec/support/payloads/pull_request_opened.json') }
+  let(:build) { Build.from_webhook(payload) }
+  let(:valid_session) { {} }
+
+  describe 'GET #index' do
+    it 'assigns all builds as @builds' do
+      build.save!
+      get(:index)
+      expect(assigns(:builds)).to eq([build])
```

Comment:
I hate testing `assigns` . And that most likely will go away on Rails 5.
Can we assert the actual response here instead?